dubbo-demo/README.md

```diff
@@ -1,29 +0,0 @@
-# Dubbo Demo
```

Comment:
I think this document is still valuable, or maybe put it in the root readme?

Reply:
@oxsean The content of this document is a little behind, later write a separate demo.